packages/backend/package.json

```diff
@@ -11,6 +11,17 @@
   },
   "main": "./dist/extension.js",
   "contributes": {
+    "configuration": {
+      "title": "AI Lab",
+      "properties": {
+        "ai-lab.models.path": {
+          "type": "string",
+          "format": "folder",
+          "default": "",
+          "description": "Custom path where to download models. Note: The extension must be restarted for changes to take effect. (Default is blank)"
```

Comment:
It's not very user friendly to need to restart ht eextension after the user changes the setting. The AI Lab extension could use the `onDidChangeConfiguration` to detect when the setting is changed, and update the `#modelsDir` at this moment. Or is there something else that needs to be initialized at init time, and cannot be updated at this moment?

Reply:
No there should be no problem at init time, we would just need to dispose the old watchers i guess. My thought was to behave similarly to the podman binary path setting which is only read at activation. As i don't think this setting will be used frequently.